code/src/components/Album.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import ArtistName from './ArtistName'
+import AlbumName from './AlbumName'
+import AlbumImage from './AlbumImage'
+
+const Album = ({ title, hrefAlbum, artist, hrefArtist, image }) => {
+    
+    return (
+        <article className="album-card">
+            <AlbumImage image={image} hrefAlbum={hrefAlbum}/>
+            <AlbumName title={title} hrefAlbum={hrefAlbum}/>
+            <ArtistName artist={artist} hrefArtist={hrefArtist}/>  
```

Comment:
Maybe could be possible to put all in one file component, and be more easy to find the info about the album.